filebeat/module/haproxy/_meta/fields.yml

```diff
@@ -7,121 +7,130 @@
       type: group
       description: >
       fields:
+
+        - name: destination
+          description: Destination information
+          type: group
+          fields:
+          - name: port
+            description: Port of the destination host
+            type: long
+
+          - name: ip
+            description: IP of the destination host
+
         - name: process_name
           description: Name of the process
 
         - name: pid
-          description: Process ID
+          description: pid of the process
```

Comment:
Use `PID`, or just left the previous description, it was fine. Or add a description about the process, something like `PID of the process that executed the request`